docs/src/main/paradox/introduction.md

```diff
@@ -44,14 +44,16 @@ Alternatively, you can bootstrap a new sbt project with Akka HTTP already
 configured using the [Giter8](http://www.foundweekends.org/giter8/) template:
 
 @@@ div { .group-scala }
-```sh
-sbt -Dsbt.version=1.2.8 new https://github.com/akka/akka-http-quickstart-scala.g8
-```
+For Scala (sbt)
+:  ```sh
+    sbt -Dsbt.version=1.3.12 new akka/akka-http-quickstart-scala.g8
```

Comment:
do we still need the `-Dsbt.version`? that only determines the sbt version that `sbt new` is using, not that the generated project is using.